README.md

```diff
@@ -111,140 +107,45 @@ It is fairly simple for a third party to constatly poll a given alias's endpoint
 
 <hr>
 
-## ENDPOINTS
-
-All endpoints (Tier 1 and Tier 2) adhere to RESTful standards and communicate using JSON. 
-
-All endpoints are served at the URL in the following format:
+## Connecting to the CAP server
 
-https://opencap.{domain}.{extension}:443/{endpoint}
+When a wallet gets an alias like john@doe.com it first has to determine which server to query.  

```

Comment:
Why doe.com:41145? Why not go straight to the SRV record? This means that requests going to ogdolo.com:41145 could potentially hang for a long time if there are firewalls/ no server

Reply:
It's a security measure.  

Let's say I want to use an own domain. I find a 3rd party server but realize that my SRV record to this server is not DNSSEC protected. I'm uncomfortable with this risk, thus I'm setting up my own server. However, if I'm using an own server the SRV records to myself would still be unprotected.  
If we instead ignore the SRV first, there's no way for a hacker to trick the wallet into connecting to the wrong server.

Reply:
Sounds good, that means that this is an edge-case. I believe that standard should be opencap.ogdolo.com:443 instead though if you don't have a SRV record

Reply:
No, that would force you to have a certificate for that subdomain.

Reply:
okay, works for me. Just make sure you explain the reasoning so that developers aren't just like "thats a random fucking port"

Reply:
It's a random default port that should not conflict with any well known service.  
I did choose 41145 because it's leetspeak of 'alias' ^^